test/Azure.Functions.Cli.Tests/LoggingFilterHelperTests.cs

```diff
@@ -1,26 +1,58 @@
 ﻿using Azure.Functions.Cli.Common;
 using Azure.Functions.Cli.Diagnostics;
 using Microsoft.Azure.WebJobs.Script;
-using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
 using System;
 using System.IO;
-using System.Text;
 using Xunit;
 
 namespace Azure.Functions.Cli.Tests
 {
     public class LoggingFilterHelperTests
     {
         private ScriptApplicationHostOptions _hostOptions;
+        private string _workerDir;
+        private string _hostJsonFilePath;
 
         public LoggingFilterHelperTests()
         {
-            _hostOptions = new ScriptApplicationHostOptions
+            try
             {
-                ScriptPath = Directory.GetCurrentDirectory()
-            };
+                _workerDir = Path.GetTempFileName();
```

Comment:
Nit: According to the [documentation](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettempfilename?view=netcore-3.1). `GetTempFileName` would create a temp file return full path. Not sure if we are using it right.

Reply:
Thanks. Skipping these tests for now. Need more time to investigate. Will address this as part of https://github.com/Azure/azure-functions-core-tools/issues/2174